docs/source/tutorial_tipsntricks.md

```diff
@@ -123,3 +123,40 @@ git repository.
 
 For more detailed instructions and features, see the
 [README](http://github.com/facebookresearch/ParlAI/blob/master/example_parlai_internal)
+
+Contributing
+------------
+
+ParlAI is maintained by a small team at Facebook, so we rely heavily on community contributions. We welcome pull requests with open wings!
```

Comment:
interestingly, I'm not sure if we ever mention that we're a team at Facebook explicitly, though I suppose the url `github.com/facebookresearch/ParlAI` gives it away

Reply:
If we'd prefer to not put ourselves in a box, I could change this to "ParlAI is maintained by a small team, ..."

Reply:
works for me! also more the truth, as we've had major contributors who never worked at Facebook